Configure

```diff
@@ -127,8 +127,7 @@ my $clang_disabled_warnings = "-Wno-unused-parameter -Wno-missing-field-initiali
 # -Wmissing-variable-declarations,
 # -Wincompatible-pointer-types-discards-qualifiers, -Wcast-align,
 # -Wunreachable-code -Wunused-parameter -Wlanguage-extension-token
-# -Wextended-offsetof
-my $clang_devteam_warn = "-Wno-unused-parameter -Wno-missing-field-initializers -Wno-language-extension-token -Wno-extended-offsetof -Qunused-arguments";
+my $clang_devteam_warn = "-Wno-unknown-warning-option -Wno-unused-parameter -Wno-missing-field-initializers -Wno-language-extension-token -Qunused-arguments";
```

Comment:
why was -Wno-extended-offsetof removed from clang_devteam_warn ?

Reply:
The starting point of this pull request was the fact that this option does not seem to be known by clang (anymore), see your pull request #9425. I searched the repository and was not quite able to locate the commit that removed it (I believe it was before 2012). But you are right, it is probably safer to keep the old option, just in case....

I'll come back to this commit as soon as I have more time again. That's why I marked it WIP and removed the 2nd reviewer label.

Reply:
Seems like the extended-offsetof group was added in Nov 2011 (https://github.com/llvm-mirror/clang/commit/b31dd8ff34903507cec1b057955f2a272ad2ee65) and removed again in Nov 2017 https://github.com/llvm-mirror/clang/commit/781caf03bdf97c1ce3fd6978b6796511408783bc